cli/dcoscli/service/main.py

```diff
@@ -190,7 +190,13 @@ def _log_task(task_id, follow, lines, file_):
     """
 
     dcos_client = mesos.DCOSClient()
-    task = mesos.get_master(dcos_client).task(task_id)
+    tasks = mesos.get_master(dcos_client).task(task_id)
+    if len(tasks) > 1:
+        msg = [("There are multiple tasks with ID matching [{}]. " +
+                "Please choose one:").format(task_id)]
+        msg += ["\t{0}".format(t["id"]) for t in tasks]
+        raise DCOSException('\n'.join(msg))
+    task = tasks[0]
```

Comment:
I think we have validation logic in a few places. Is there any way we can unify this? Can you please provide example output for both commands so I understand what we are trying to do?